src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/ManageApprenticesViewModel.cs

```diff
@@ -25,7 +25,7 @@ public void SortedByHeader()
             }
         }
 
-        public ManageApprenticesFilterModel FilterModel { get; set; }
-        public bool ShowPageLinks  => FilterModel.TotalNumberOfApprenticeshipsFound > ProviderCommitmentsWebConstants.NumberOfApprenticesPerSearchPage;
+        public ApprenticesFilterModel FilterModel { get; set; }
+        public bool ShowPageLinks  => FilterModel.TotalNumberOfApprenticeshipsFound > Constants.ApprenticesSearch.NumberOfApprenticesPerSearchPage;
     }
```

Comment:
As previous comment about naming, viewmodel names should align with the request names, so this would be an `IndexViewModel`, and can be moved into the `Apprentice` folder to avoid confusion and namespace collisions.

Reply:
Moved and renamed